bin/create.js

```diff
@@ -302,26 +337,76 @@ function handleCreate (argv) {
                 }
             })
             .then(function (profile) {
-                return profile.createToken(argv)
+                var promises = [];
+                var unnamed_url = profile.url + '/api/run/'
+                    + (argv.container || profile.container);
+
+                if (argv.name)
+                    var named_url = profile.url
+                        + '/api/run/'
+                        + (argv.container || profile.container)
+                        + '/' + argv.name;
+
+                if(argv.share) {
+                    var aud = named_url.replace('https:\/\/', '');
```

Comment:
If argv.share is present, is argv.name also ALWAYS present? If not, the named_url here will be undefined.